.common-ci.yml

```diff
@@ -28,6 +28,7 @@ stages:
   - test
   - lint
   - helm-lint
+  - helm
```

Comment:
we should publish helm chart after the image has been successfully published. i.e this stage should be part of release.

Reply:
Done, fixed